tests/e2e_tests/test_user_creation.py

```diff
@@ -38,6 +38,7 @@ async def test_create_admin_user(test_async_client):
     profile = UserProfile(
         username=username,
         hashed_password=hashed_password,
+        email='test-admin@gmail.com',
```

Comment:
It might be better to use `@kernelci.org` email addresses, just in case some code runs this and actually tries to send emails to some GMail address that might really exist.  But that's really a corner-case thing.

Reply:
Okay. Yes, I agree with your point. I'll update this and other tests as well. Thanks.